akka-stream/src/main/scala/akka/stream/javadsl/Flow.scala

```diff
@@ -207,20 +207,34 @@ object Flow {
 
   /**
    * Creates a real `Flow` upon receiving the first element. Internal `Flow` will not be created
-   * if there are no elements, because of completion or error.
-   * The materialized value of the `Flow` will be the materialized
-   * value of the created internal flow.
+   * if there are no elements, because of completion, cancellation, or error.
    *
-   * If `flowFactory` throws an exception and the supervision decision is
-   * [[akka.stream.Supervision.Stop]] the materialized value of the flow will be completed with
-   * the result of the `fallback`. For all other supervision options it will
-   * try to create flow with the next element.
+   * The materialized value of the `Flow` is the value that is created by the `fallback` function.
    *
-   * `fallback` will be executed when there was no elements and completed is received from upstream
-   * or when there was an exception either thrown by the `flowFactory` or during the internal flow
-   * materialization process.
+   * '''Emits when''' the internal flow is successfully created and it emits
    *
-   * Adheres to the [[ActorAttributes.SupervisionStrategy]] attribute.
+   * '''Backpressures when''' the internal flow is successfully created and it backpressures
+   *
+   * '''Completes when''' upstream completes and all elements have been emitted from the internal flow
+   *
+   * '''Cancels when''' downstream cancels
+   */
+  @deprecated("Use lazyInitAsync instead.", "2.5.12")
```

Comment:
perhaps include a reason: ", because materialized value of this method is invalid"

Reply:
👍